### ISSUE TITLE
Cinnamon panel icons squished, text not readable

### ISSUE THREAD

**Issue by 392781** [EliverLara/Juno#34]
Icons and text on Cinnamon are broken:

![image](https://user-images.githubusercontent.com/25296775/124524483-0ef74200-ddb0-11eb-8d8c-019a6bef39d9.png)

Icons are squished, the background to the icons is black, and the text is greyed out and not readable.

**Comment by EliverLara** [comment]
Hi, Are you using some extension to get transparent panels?

**Comment by 392781** [comment]
Yes.  The issues still remain without the transparency though.

**Comment by EliverLara** [comment]
Could you tell me what extension are you using? because I'm not able to reproduce the issue on my side:

![Captura de pantalla de 2021-07-07 13-54-20](https://user-images.githubusercontent.com/9838260/124813949-0aa36400-df2b-11eb-89ee-ece8607d9967.png)

**Comment by 392781** [comment]
Yes, I'll send more details in a few hours once I get back to my machine.

**Comment by 392781** [comment]
I'm using "Transparent panels" and "Cinnamon Maximus" (both Cinnamon extensions).

**Comment by 392781** [comment]
I went looking through the code.  I'm not very well versed with Linux theming so bear with me...

Addressing the icon issue: I am using the "Grouped Window List" applet for my panel.  The issue of icon "squishing" occurs when the panel "Colored icon size" or the "Panel height" is changed below a threshold.  Here is an image of the icons using 32px colored icon size and 40px panel height:

![image](https://user-images.githubusercontent.com/25296775/126252517-5dc49aa2-6df7-457d-a907-37f291a74061.png)

When I adjust the panel height below 40px (35px in this case), the following happens (icon squishing is present):

![image](https://user-images.githubusercontent.com/25296775/126252591-69e0f903-8659-4366-a275-f7992f6e5136.png)

I'm guessing this occurs due to having oversized icons within too small of an asset/container.

Addressing the icon background issue on transparent panels: I tested out changing elements of `.grouped-window-list-item-box`.  Adding an alpha channel to the `background-color` is unsupported as any bit of transparency immediately turns off the black background (along with the yellow highlighting).  At the same time, this also "fixes" the icon squishing issue.  I'm guessing that one of the assets restricts the icon width after a certain height and also is not inherently transparent.  

This squishing and transparency issue runs across your other themes as well.  The transparency is likely a stylistic choice, but the squishing at more restricting panel sizes can probably be fixed.  I'd be happy to make adjustments if you point me in the right direction.  If anything, I hope that this is useful information for the future.
